paint/PAINT_DrawWindow.h

```diff
@@ -38,9 +38,10 @@ namespace paint
 		DrawWindow& operator=(const DrawWindow& that) = delete;
 		DrawWindow& operator=(DrawWindow&& that) = delete;
 
+		bool mouseEnter(bool clicked = false) override;
```

Comment:
Just a comment. In future we might have the possibility of using the right mouse button to draw background colours whilst the left draws foreground. So a boolean woulndn't suffice here.